benchmark/fs/bench-timesSync.js

```diff
@@ -0,0 +1,65 @@
+'use strict';
+
+const common = require('../common');
+const fs = require('fs');
+const tmpdir = require('../../test/common/tmpdir');
+tmpdir.refresh();
+
+const bench = common.createBenchmark(main, {
+  type: ['existing', 'non-existing'],
+  func: [ 'utimes', 'futimes', 'lutimes' ],
+  n: [1e3],
+});
+
+function main({ n, type, func }) {
+  const useFds = func === 'futimes';
+  const fsFunc = fs[func + 'Sync'];
+
+  const time = Date.now() / 1000;
+
+  switch (type) {
+    case 'existing': {
+      const files = [];
+
+      // Populate tmpdir with mock files
+      for (let i = 0; i < n; i++) {
+        const path = tmpdir.resolve(`timessync-bench-file-${i}`);
+        fs.writeFileSync(path, 'bench');
+        files.push(useFds ? fs.openSync(path, 'r+') : path);
+      }
+
+      bench.start();
+      for (let i = 0; i < n; i++) {
+        fsFunc(files[i], time, time);
+      }
+      bench.end(n);
+
+      if (useFds) files.forEach((x) => {
+        try {
+          fs.closeSync(x);
+        } catch {
+          // do nothing
+        }
+      });
+
+      break;
+    }
+    case 'non-existing': {
+      const file = useFds ? (1 << 30) : tmpdir.resolve(`.non-existing-file-${Date.now()}`);
```

Comment:
Can you move this line inside the for loop, and use the iterator - i, instead of Date.now?